crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -435,14 +457,17 @@ impl Entities {
     pub fn resolve_from_id(&self, id: u32) -> Option<Entity> {
         let idu = id as usize;
```

Comment:
I'm not really a fan of the `idu` name here, however I don't have a good suggestion for anything different.

Reply:
All that comes to mind is:
- idx / index
- slot_id / slot_idx / slot_index

Reply:
In the last push, this is the only comment I don't think I've addressed